services/app-web/src/hooks/useDocument.tsx

```diff
@@ -0,0 +1,42 @@
+import { Document } from '../gen/gqlClient'
+import { useS3 } from '../contexts/S3Context'
+import { BucketShortName } from '../s3/s3Amplify'
+import { SubmissionDocument } from '../common-code/healthPlanFormDataType'
+
+type UnionDocumentType = Document | SubmissionDocument
+type ParsedDocumentWithLinkType =
+    | ({ url: string | null } & Document)
+    | SubmissionDocument
+
+const useDocument = () => {
```

Comment:
is this stuff merged in Hana’s earlier PR or is it new with this PR? You might want to do a merge from main

Reply:
This is new from this PR.